include/wlr/types/wlr_linux_dmabuf_v1.h

```diff
@@ -56,7 +56,27 @@ struct wlr_linux_dmabuf_feedback_v1 {
 struct wlr_linux_dmabuf_feedback_v1_tranche {
 	dev_t target_device;
 	uint32_t flags; // bitfield of enum zwp_linux_dmabuf_feedback_v1_tranche_flags
-	struct wlr_drm_format_set formats;
+	struct wl_array format_indices;
+};
+
+/*
+ * This is one entry in the format list we send the client in the format_table
+ * event.
+ */
+struct wlr_dmabuf_format_table_entry {
+    uint32_t format;
+    uint32_t pad; /* unused */
+    uint64_t modifier;
+};
+
+struct wlr_dmabuf_format_table {
+    int fd;
+    size_t len;
+    /* This will be mmapped from the fd above */
+    struct wlr_dmabuf_format_table_entry *data;
+    /* The default indices list. datatype is uint16_t */
+	struct wl_array indices;
```

Comment:
Hm, I think it would be better to remove this, and just use `wlr_linux_dmabuf_feedback_v1_tranche`?